src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -137,6 +139,11 @@ type SlotConfig =
             ApplyIPSecurityRestrictionsToScm = this.ApplyIPSecurityRestrictionsToScm
             EnablePublicNetworkAccess = this.EnablePublicNetworkAccess
             ZipDeployPath = None
+            NetFrameworkVersion = this.Runtime |> Option.map (fun r -> r |> function
```

Comment:
The joy of currying means you don't need to declare the lambda here, you can just use `function` directly which has a "dangling" second argument which is left to `Option.map` to fill in.
```suggestion
            NetFrameworkVersion = this.Runtime |> Option.map (function
```